packages/discovery-provider/src/tasks/index_trending.py

```diff
@@ -123,6 +123,24 @@ def index_trending(self, db: SessionManager, redis: Redis, timestamp):
                         f"index_trending.py | Cached trending ({version.name} version) \
                         for {genre}-{time_range} in {total_time} seconds"
                     )
+            # Cache premium tracks
+            cache_start_time = time.time()
+            res = generate_unpopulated_trending_from_mat_views(
+                session=session,
+                genre=genre,
+                time_range="week",
+                strategy=strategy,
+                usdc_purchase_only=True,
+            )
+            key = make_trending_tracks_cache_key("week", None, strategy.version)
```

Comment:
yea true but even a track from a verified user probably won't show up on trending within the first hour. alternative i was thinking would be to just modify the ranking to something very simple because trending calculations are just really intensive

Reply:
Oh I guess I didn't realize the premium tracks explore page used "trending" logic. But I guess it wouldn't make sense for it to just be a reverse chronological list of premium tracks.